Common/include/toolboxes/solution_toolbox.hpp

```diff
@@ -0,0 +1,1084 @@
+/*!
+ * \file solution_toolbox.hpp
+ * \brief Header file for the verification solution classes.
+ *        The implementations are in the <i>solution_toolbox.cpp</i> file.
+ * \author T. Economon, E. van der Weide
+ * \version 6.2.0 "Falcon"
+ *
+ * The current SU2 release has been coordinated by the
+ * SU2 International Developers Society <www.su2devsociety.org>
+ * with selected contributions from the open-source community.
+ *
+ * The main research teams contributing to the current release are:
+ *  - Prof. Juan J. Alonso's group at Stanford University.
+ *  - Prof. Piero Colonna's group at Delft University of Technology.
+ *  - Prof. Nicolas R. Gauger's group at Kaiserslautern University of Technology.
+ *  - Prof. Alberto Guardone's group at Polytechnic University of Milan.
+ *  - Prof. Rafael Palacios' group at Imperial College London.
+ *  - Prof. Vincent Terrapon's group at the University of Liege.
+ *  - Prof. Edwin van der Weide's group at the University of Twente.
+ *  - Lab. of New Concepts in Aeronautics at Tech. Institute of Aeronautics.
+ *
+ * Copyright 2012-2019, Francisco D. Palacios, Thomas D. Economon,
+ *                      Tim Albring, and the SU2 contributors.
+ *
+ * SU2 is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * SU2 is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with SU2. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <cmath>
+#include "../config_structure.hpp"
+
+/*!
+ * \class CVerificationSolution
+ * \brief Class for holding verification PDE solutions, e.g., phi = phi(x,y,z,t),
+ *        used for initial conditions, analytic solutions, manufactured solutions.
+ * \author T. Economon, E. van der Weide
+ */
+class CVerificationSolution {
```

Comment:
Since we start reorganizing the file structure soon, it would be maybe good to already start with that here. Separating all classes into individual header/source files inside of an MMS directory. What do you say ?

Reply:
Fine with me. Do you want that MMS directory then directly in Common/include or in Common/include/toolboxes? Same for the sources of course.

Reply:
We can put the MMS directory in the toolboxes directory. I don't mind having additional sub-directories.